data/darktableconfig.xml.in

```diff
@@ -200,6 +200,38 @@
     <shortdescription>enable disk backend for full preview cache</shortdescription>
     <longdescription>if enabled, write full preview to disk (.cache/darktable/) when evicted from the memory cache.\nnote that this can take a lot of memory (several gigabytes for 20k images) and will never delete cached thumbnails again.\nit's safe though to delete these manually, if you want.\nlight table performance will be increased greatly when zooming image in full preview mode.</longdescription>
   </dtconfig>
+  <dtconfig prefs="processing" section="cpugpu" restart="true">
```

Comment:
BTW, I'm not sure we need a restart. All the values there are read while processing.

Reply:
Changing this at runtime would require difficult stuff in the preference-has-changed callback to make sure the database searching for "newer than last check" works. (we have to reset startpoint to zero)

We could do otherwise but that would mean permanent file accesses.

Reply:
Ok.